runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersListView/UsersListView.tsx

```diff
@@ -62,7 +62,7 @@ const UsersListView: FC = () => {
     };
 
     useEffect(() => {
-        dispatch(tenantsActions.getAll());
+        hasAdminAccess && dispatch(tenantsActions.getAll());
```

Comment:
thinking of the future I think it's good to rename booleans like that to hasCrossTenantAccess or sth

Reply:
cause now we have two admins

Reply:
why add this condition if TENANT_ADMIN has access to this tab too? The difference is that TENANT_ADMIN sees users in his/hers tenant, and ADMIN sees all users across all tenants.

But the refresh should happen for both of them, or am I wrong?

Reply:
It has, but this endpoint is for retrieving all tenants. in Users tab tenant admin do not see the list of tenants to choose from and in the table there i no tenant column (as all already is within the tenant)